wpilib/wpilib/spi.py

```diff
@@ -193,7 +193,7 @@ def setChipSelectActiveLow(self) -> None:
         """Configure the chip select line to be active low."""
         hal.setSPIChipSelectActiveLow(self.port)
 
-    def write(self, dataToSend: bytes) -> int:
+    def write(self, dataToSend: Union[bytes, List[int]]) -> int:
```

Comment:
If we're going to do this, shouldn't this be `Iterable[int]`? Or do we not want to guarantee that? If not, `Sequence[int]` would be more appropriate.